### ISSUE TITLE
Container will cause huge amount of traffic to github downloading config.zip files when misconfigured (in my case 75 GB of traffic in 12 hours)

### ISSUE THREAD

**Issue by dennisvanderpool** [haugene/docker-transmission-openvpn#2613]
### Is there a pinned issue for this?

- [X] I have read the pinned issues and could not find my issue

### Is there an existing or similar issue/discussion for this?

- [X] I have searched the existing issues
- [X] I have searched the existing discussions

### Is there any comment in the documentation for this?

- [X] I have read the documentation, especially the FAQ and Troubleshooting parts

### Is this related to a provider?

- [X] I have checked the provider repo for issues
- [X] My issue is NOT related to a provider

### Are you using the latest release?

- [X] I am using the latest release

### Have you tried using the dev branch latest?

- [ ] I have tried using dev branch

### Docker run config used

docker run \
        -d \
        --cap-add=NET_ADMIN \
        --name transmission-openvpn \
        --hostname=transmission \
        -v /var/lib/docker/volumes/samba_data/_data/media/Downloads/Torrents:/data \
        -v /etc/localtime:/etc/localtime:ro \
        -e CREATE_TUN_DEVICE=true \
        -e OPENVPN_PROVIDER=PUREVPN \
        -e OPENVPN_CONFIG=vleu-no2-ovpn-udp \
        -e OPENVPN_USERNAME=myincorrectusername \
        -e OPENVPN_PASSWORD=myincorrectpassword \
        -e WEBPROXY_ENABLE=false \
        -e LOCAL_NETWORK=192.168.1.0/24 \
        --log-driver json-file \
        --log-opt max-size=10m \
        -p 9091:9091 \
        --restart=unless-stopped \
        haugene/transmission-openvpn:latest

### Current Behavior

When connection to VPN fails it will download zipfile with configs from github, but with this new zip it still fails, it restarts and downloads zip again. This causes 75 GB of zip downloads in about 12 hours.

Please note that I have the container running on multiple servers and switched VPN provider. So this was a misconfiguration which I did not notice/needed, but instead of this container just failing and creating some log files it download 75 GB of config zip files. 

Maybe the container should be changed in such a way it won't cause huge log file or downloads when it's failing or sleep in between retries.

### Expected Behavior

Zip file is not download over and over again, but only when needed.

### How have you tried to solve the problem?

I will solve it by for now not running the container. It won't be a problem if I fix the configuration (just copying it from a working server).

### Log output

Creating TUN device /dev/net/tun
Using OpenVPN provider: PUREVPN
Running with VPN_CONFIG_SOURCE auto
No bundled config script found for PUREVPN. Defaulting to external config
Downloading configs from https://github.com/haugene/vpn-configs-contrib/archive/main.zip into /tmp/tmp.Ppqzp6Hs96

### HW/SW Environment

```markdown
- OS: Linux server3 6.3.1-arch2-1 #1 SMP PREEMPT_DYNAMIC Wed, 10 May 2023 08:54:47 +0000 x86_64 GNU/Linux
- Docker: Docker version 23.0.6, build ef23cbc431
```

### Anything else?

Thanks for the software :-)

**Comment by pkishino** [comment]
already implemented in #2592 
will be merged in soon

**Comment by dennisvanderpool** [comment]
@pkishino Thanks! 😀
